reladomo/src/main/java/com/gs/fw/common/mithra/finder/NonPrimitiveInOperation.java

```diff
@@ -326,7 +328,11 @@ public Timestamp getSetValueAsTimestamp(int index)
     @Override
     protected void appendSetToString(ToStringContext toStringContext)
     {
-        toStringContext.append(this.set.toString());
+        String quotedStrings = SetAdapter.adapt(this.set)
+                .asLazy()
+                .collect(each -> "\"" + each + "\"")

```

Comment:
This now applies to far more than just strings. It'll put quotes around Date, Timestamp, ByteArray, BigDecimal and Time. It's probably better to add a method to ```NonPrimitiveAttribute```, e.g. ```String formattedValue(Object)```, which might also clean up some of the other code like ```AtomicEqualityOperation.zToString```

Reply:
In TestAtomicOperationsToString.testInOperation() you can see that it does affect Dates and Timestamps. It looks like it doesn't affect byte arrays. I think BigDecimal and Time might be uncovered by tests, but I can check. Of the types that are covered, testInOperation() is now consistent with testEqualsOperation() which was my intention. What do you think?

Reply:
So I'm pretty sure I don't want quotes around BigDecimal and ByteArray. I'm ambivalent about quotes for Date/Timestamp/Time -- happy to keep them because that's what's already there.

Maybe add tests for BigDecimal `eq`, `greaterThan`, `in` and see if it'll simplify things if we delegate back to the attribute to format the value in multiple places.